way_to_home/place/models.py

```diff
@@ -1 +1,68 @@
 """This module implements class that represents the place entity."""
+
+from user.models import User
+from django.db import models, IntegrityError
+from utils.abstract_models import AbstractModel
```

Comment:
```suggestion
from user.models import User
from django.db import models, IntegrityError
from utils.abstract_models import AbstractModel
```
Optimize imports, please